platformio.ini

```diff
@@ -1121,18 +1121,14 @@ build_flags   = ${env:chitu_f103.build_flags} -DCHITU_V5_Z_MIN_BUGFIX
 # Creality (STM32F103RET6)
 #
 [env:STM32F103RET6_creality]
-platform        = ${common_stm32f1.platform}
-extends         = common_stm32f1
-board           = genericSTM32F103RE
-build_flags     = !python Marlin/src/HAL/STM32F1/build_flags.py
-  ${common.build_flags} -std=gnu++14 -DSTM32_XL_DENSITY -DTEMP_TIMER_CHAN=4
```

Comment:
You may notice that I removed `STM32_XL_DENSITY`. That is because this is _not_ an XL Density chip, it is a high density chip.